uint32 status
float32 vel_res
float32 pos_x
float32 pos_y
